doc/methods/reprocessing.rst

```diff
@@ -3,4 +3,162 @@
+processes is defined in a DOT file. The process graph must be directed and
+acyclic in order to work with SaltProc. At runtime,
+SaltProc reads this input file to create :class:`saltproc.Process` objects for each item
+in the file. Every process file must have at least a :class:`core_outlet` and
+:class:`core_inlet` Process.
+
+Material reprocessing
+---------------------
+
+Recall that SaltProc uses a *batchwise* reprocessing scheme.
+Let :math:`\mathbf{n}(t)^{j}` denote the nuclide mass vector for depletable material
+:math:`j` as a function of time. For each depletable material :math:`j`, the depletion
+solver numerically integrates the equation
+
+.. math::
+
+    \frac{d\mathbf{n}^{j}(t)}{dt} = \mathbf{A}(\mathbf{n}^{j}(t), t)
+
+from time :math:`i` to time :math:`i+1` to get :math:`\mathbf{n}^{j}(i+1)`, where :math:`A` is the
+depletion matrix. This sequence is called a **depletion step**.
+
+Now, let the mass and volume of depletable material :math:`j` be
+:math:`m^{j}` and :math:`V^{j}` respectively. Let the mass flow rate of process :math:`p` be
+:math:`\dot{m}_{p}`. At the end of each depletion step, SaltProc constructs process
+paths from the process graph defined in the DOT file, and sequentially applies
+each process :math:`p` in each path :math:`r` to the relevant materials to obtain through
+and waste streams for each material. SaltProc tracks the mass and nuclide vector for both through and waste streams. For through
+streams, SaltProc also tracks the volume and mass flow rate. For every node
+:math:`p\in[0,l]` where :math:`0` represents the core outlet and :math:`l` represents the core
+inlet, in the path :math:`r`, for the through streams we have
+
+.. math::
+
+    \mathbf{n}^{j}_{\text{through, }p,r} = \mathbf{n}^{j}_{\text{through, }p-1,r} (1 - \pmb{\epsilon}^{j}_{p,r})
+
+.. math::
+
+    m^{j}_{\text{through, } p,r} = \alpha_{p} m^{j}_{\text{through, }p-1,r} - m^{j}_{\text{waste, }p,r}
+
+.. math::
+
+    V^{j}_{\text{through, }p,r} = \alpha_{p}V^{j}_{\text{through, }p-1,r}
+
+where 
+
+.. math::
+
+    \alpha_{p,r} = \frac{\dot{m}_{p,r}}{\dot{m}_{\text{outlet}}}
+
+and the initial conditions are 
+
+.. math::
+
+    \mathbf{n}^{j}_{\text{through, }0,r} = \mathbf{n}^{j}(i+1)
+
+.. math::
+
+    m^{j}_{\text{through, }0,r} = \rho^{j}(i+1)V^{j}(i+1)
+
+.. math::
+
+    V^{j}_{\text{through, }0,r} = V^{j}(i+1)
+
+Similarly, for the waste streams, we have
+
+.. math::
+
+    \mathbf{n}^{j}_{\text{waste, }p,r} = \mathbf{n}^{j}_{\text{through, }p-1,r} \cdot \pmb{\epsilon}^{j}_{p,r}
+
+.. math::
+
+    m^{j}_{\text{waste, }p,r} = \alpha_{p,r} m^{j}_{\text{through, }p-1,r} \langle\mathbf{1},\mathbf{n}^{j}_{\text{waste, }p,r}\rangle
+
+
+SaltProc does not currently track the volume and mass flow rate of waste streams.
+In practice, since it is extremely resource intensive to separate individual
+isotopes, SaltProc only allows extraction efficiencies to be defined for
+elements. So, for any isotope :math:`a` of xenon,
```

Comment:
You could also mention here that for MSRs, chemical processes operate on elements, so having isotope specific extraction efficiencies is not required